### DISCUSSION TITLE
a potential error with the documentation ?

### DISCUSSION THREAD

**Discussion by hugzenfeu** [remix-run/remix#9442] [General]
I'm trying to learn Remix, and I believe I've identified a potential mistake in the code snippet provided at the following link: https://remix.run/docs/en/main/discussion/form-vs-fetcher#creating-a-new-record

It seems that there might be a missing default to the export on line 21. Can someone confirm if this is correct, or if I'm overlooking something?

**Comment by kiliman** [comment]
Yes, it should be a `default export`. In addition, `useActionData()` will return `undefined` during the initial load (the value is only present after the action is called). So dereferencing `{errors}` will throw an error. 

```ts
default export function NewRecipe() {
  const { errors } = useActionData<typeof action> ?? {} // guard against destructuring undefined
  // ...
```
